src/Traits/BinaryFlags.php

```diff
@@ -196,4 +201,120 @@ public function checkAnyFlag($mask)
     {
         return $this->checkFlag($mask, false);
     }
+
+    /**
+     * Return the current element
+     *
+     * @link  https://php.net/manual/en/iterator.current.php
+     * @return mixed Can return any type.
+     * @since 5.0.0
```

Comment:
This will be a feature for `v1.2.0` of binary flags, so maybe add it as `@since 1.2.0`? as well for the `@since` statements below